app/src/main/java/com/naxtlevelofandroiddevelopment/marvelgallery/network/MarvelApi.kt

```diff
@@ -0,0 +1,17 @@
+package com.naxtlevelofandroiddevelopment.marvelgallery.network
```

Comment:
Presenters are framework independent they shouldn't stay in the same package with activities, in the extreme they can be moved to a separate module without Android FW dependency. So no, definitely `presenter` package should be there, the naming of the rest layers is up to you. It can be `view` `presenter` `data`.